sdk/communication/communication-sms/CHANGELOG.md

```diff
@@ -6,17 +6,23 @@
 
 - `SmsClient` added a constructor that supports `TokenCredential`.
 
+### Breaking Changes
```

Comment:
I had an `###Added` section with these points for C#: 
- Added support for 1:N SMS messaging.
- Added support for tagging SMS messages.
- Send method series in SmsClient are idempotent under retry policy.